io-pay-portal-fe/src/api/client.ts

```diff
@@ -9,19 +9,19 @@ import { createClient } from "../../generated/client";
 
 import { getConfig } from "../util/config";
 
-// 5 seconds timeout by default
-const DEFAULT_REQUEST_TIMEOUT_MS = 5000;
-
 // Must be an https endpoint so we use an https agent
 const abortableFetch = AbortableFetch(agent.getHttpFetch(process.env));
 const fetchWithTimeout = toFetch(
-  setFetchTimeout(DEFAULT_REQUEST_TIMEOUT_MS as Millisecond, abortableFetch)
+  setFetchTimeout(
+    getConfig("IO_PAY_PORTAL_API_REQUEST_TIMEOUT") as Millisecond,
```

Comment:
it may be useful to set a default value here.
In the absence of the env variables, as it happened to me, because I hadn't  done the `export`

Reply:
I would add a script in `package.json`  ( ex. `start` ) with all steps needed to build and start dev server `yarn generate && yarn build && yarn dev`

Reply:
`IO_PAY_PORTAL_API_REQUEST_TIMEOUT` is required in the configurations defined in `IConfig`